src/components/Authenticated.js

```diff
@@ -62,11 +62,19 @@ export default function Authenticated(props) {
 
             <h3>ID Token claims</h3>
             <pre
-              className="json-container"
-              dangerouslySetInnerHTML={{
-                __html: prettyPrintJson.toHtml(decodeToken(id_token)),
+                className="json-container"
+                dangerouslySetInnerHTML={{
+                  __html: prettyPrintJson.toHtml(decodeToken(id_token)),
               }}
             />
+            <h3>ID Token claims as a string</h3>
+            <pre
+                id="tokenClaims"
```

Comment:
FYI: access token is more important than an ID token as it will be used as a bearer token.